apis/palette_api/app/api.py

```diff
@@ -18,27 +17,21 @@
     docs_url='/docs',
     redoc_url='/redoc'
 )
-# CORS
```

Comment:
How come this has been removed?

Reply:
no idea when/how that was removed... it's back now